docs/source/cross_validation.rst

```diff
@@ -4,9 +4,15 @@ Cross Validation
 See the `scikit-learn cross validation documentation`_ for a fuller discussion of cross validation.
 This document only describes the extensions made to support Dask arrays.
 
-The simplest way to split one or more Dask arrays is with :func:`dask_ml.model_selection.train_test_split`.
+The simplest way to split one or more Dask arrays is with :func:`dask_ml.model_selection.train_test_split`:
+
+.. autosummary::
+   dask_ml.model_selection.train_test_split
+
+Here's an example usage:
 
 .. ipython:: python
+   :okwarning:
```

Comment:
What's the warning here?

Reply:
With NumPy 1.18.1:

``` 
>>>-------------------------------------------------------------------------
Warning in /Users/scott/Developer/stsievert/dask-ml/docs/source/cross_validation.rst at block ending on line 22
Specify :okwarning: as an option in the ipython:: block to suppress this message
----------------------------------------------------------------------------
/Users/scott/anaconda3/envs/dask-ml-test/lib/python3.7/importlib/_bootstrap.py:219: RuntimeWarning: numpy.ufunc size changed, may indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
  return f(*args, **kwds)
<<<-------------------------------------------------------------------------
```

Reply:
OK can you remove this okwarning then? That sounds like an issue with either your local install or our CI script.

Ideally okwarning is only used when we knowingly emit a warning.